test/SlowTests/Issues/RavenDB-19109.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using FastTests;
+using Raven.Client.ServerWide.Operations;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_19109 : RavenTestBase
+{
+    public RavenDB_19109(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    private record Item(string Name);
+
+    [Fact]
+    public void DatabaseThrowsOnOpenWhenDisableMarkerIsInDirectory()
```

Comment:
We need similar test for disabling an index. It gets faulty in that case and after deleting the marker we can open it again.